samples/dotnet-runtimeinfo/Program.cs

```diff
@@ -5,34 +5,26 @@
 using static System.Console;
 using static System.IO.File;
 
-namespace RuntimeEnvironment
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            WriteLine("**.NET information");
-            WriteLine($"{nameof(Environment.Version)}: {Environment.Version}");
-            WriteLine($"{nameof(RuntimeInformation.FrameworkDescription)}: {RuntimeInformation.FrameworkDescription}");
-            var assemblyInformation = ((AssemblyInformationalVersionAttribute[])typeof(object).Assembly.GetCustomAttributes(typeof(AssemblyInformationalVersionAttribute), false))[0];
-            var informationalVersionSplit = assemblyInformation.InformationalVersion.Split('+');
-            WriteLine($"Libraries version: {informationalVersionSplit[0]}");
-            WriteLine($"Libraries hash: {informationalVersionSplit[1]}");
-            WriteLine();
-            WriteLine("**Environment information");
-            WriteLine($"{nameof(RuntimeInformation.OSDescription)}: {RuntimeInformation.OSDescription}");
-            WriteLine($"{nameof(Environment.OSVersion)}: {Environment.OSVersion}");
-            WriteLine($"{nameof(RuntimeInformation.OSArchitecture)}: {RuntimeInformation.OSArchitecture}");
-            WriteLine($"{nameof(Environment.ProcessorCount)}: {Environment.ProcessorCount}");
-            WriteLine();
+var assemblyInformation = ((AssemblyInformationalVersionAttribute[])typeof(object).Assembly.GetCustomAttributes(typeof(AssemblyInformationalVersionAttribute), false))[0];
```

Comment:
This seems to break the rule of using var only when it's obvious - rule #10 at 
https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md

Reply:
Good call. Fixed.